timesketch/lib/sigma_util.py

```diff
@@ -245,3 +258,61 @@ def get_sigma_rule(filepath, sigma_config=None):
             {'file_relpath':file_relpath})
 
         return rule_return
+
+def get_sigma_rule_by_text(rule_text, sigma_config=None):
+    """ Returns a JSON represenation for a rule
+
+    Args:
+        rule_text: Text of the sigma rule to be parsed
+        sigma_config: config file object
+
+    Returns:
+        Json representation of the parsed rule
```

Comment:
I see a bunch of `return None`

Reply:
All the ```return None``` have been removed and proper ```Raises``` has been docstringed